apps/web/src/features/agent/useCrew.ts

```diff
@@ -0,0 +1,174 @@
+import { useEffect, useMemo, useState } from "react"
+import { useArrayRef } from "~core/components/hooks/useArrayRef"
+import { useWindowAI } from "~core/components/hooks/useWindowAI"
+import {
+  useAgentManager,
+  type AgentConfig
+} from "~core/providers/agent-manager"
+import { useLog } from "~features/agent/useLog"
+
+const getSystemPrompt = (goal: string, ac: AgentConfig) =>
```

Comment:
nit (not needed for mvp): might be cool to let users inspect and edit these prompts from the ui

Reply:
Actually, we can make this a feature of the window extension! #60